<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="7" passed="5" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-02-03T08:23:43Z" name="Suite" finished-at="2020-02-03T08:24:39Z" duration-ms="55908">
    <groups>
    </groups>
    <test started-at="2020-02-03T08:23:43Z" name="Test" finished-at="2020-02-03T08:24:39Z" duration-ms="55908">
      <class name="com.apex.ecommerce.order.test.OrderTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.order.test.OrderTest@6a62689d]" started-at="2020-02-03T00:23:43Z" name="setUp" finished-at="2020-02-03T00:23:49Z" duration-ms="6039" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validatePidTest()[pri:0, instance:com.apex.ecommerce.order.test.OrderTest@6a62689d]" started-at="2020-02-03T00:23:49Z" name="validatePidTest" finished-at="2020-02-03T00:24:11Z" duration-ms="22165" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validatePidTest -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.order.test.OrderTest@6a62689d]" started-at="2020-02-03T00:24:11Z" name="close" finished-at="2020-02-03T00:24:11Z" duration-ms="148" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.apex.ecommerce.order.test.OrderTest -->
      <class name="com.apex.ecommerce.registration.test.RegistrationTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.registration.test.RegistrationTest@60fa3495]" started-at="2020-02-03T00:24:31Z" name="setUp" finished-at="2020-02-03T00:24:34Z" duration-ms="2614" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testCreateAccount()[pri:0, instance:com.apex.ecommerce.registration.test.RegistrationTest@60fa3495]" started-at="2020-02-03T00:24:34Z" name="testCreateAccount" finished-at="2020-02-03T00:24:39Z" duration-ms="4999" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateAccount -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.registration.test.RegistrationTest@60fa3495]" started-at="2020-02-03T00:24:39Z" name="close" finished-at="2020-02-03T00:24:39Z" duration-ms="170" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.apex.ecommerce.registration.test.RegistrationTest -->
      <class name="com.apex.ecommerce.signon.test.SignonTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:11Z" name="setUp" finished-at="2020-02-03T00:24:14Z" duration-ms="3027" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithAllBlanks()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:14Z" name="testWithAllBlanks" finished-at="2020-02-03T00:24:15Z" duration-ms="693" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithAllBlanks -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:15Z" name="close" finished-at="2020-02-03T00:24:15Z" duration-ms="144" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:15Z" name="setUp" finished-at="2020-02-03T00:24:18Z" duration-ms="3051" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithBlankUserId()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:18Z" name="testWithBlankUserId" finished-at="2020-02-03T00:24:19Z" duration-ms="708" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithBlankUserId -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:19Z" name="close" finished-at="2020-02-03T00:24:19Z" duration-ms="154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:19Z" name="setUp" finished-at="2020-02-03T00:24:21Z" duration-ms="2393" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithInvalidCharacters()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:21Z" name="testWithInvalidCharacters" finished-at="2020-02-03T00:24:22Z" duration-ms="642" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.apex.ecommerce.signon.test.SignonTest.testWithInvalidCharacters(SignonTest.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithInvalidCharacters -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:23Z" name="close" finished-at="2020-02-03T00:24:23Z" duration-ms="125" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:23Z" name="setUp" finished-at="2020-02-03T00:24:26Z" duration-ms="2996" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithSuccessCredentials()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:26Z" name="testWithSuccessCredentials" finished-at="2020-02-03T00:24:26Z" duration-ms="785" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithSuccessCredentials -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:26Z" name="close" finished-at="2020-02-03T00:24:27Z" duration-ms="239" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:27Z" name="setUp" finished-at="2020-02-03T00:24:30Z" duration-ms="2880" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithWrong()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:30Z" name="testWithWrong" finished-at="2020-02-03T00:24:31Z" duration-ms="980" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [valid] but found [Invalid Username / Password!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [valid] but found [Invalid Username / Password!]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.apex.ecommerce.signon.test.SignonTest.testWithWrong(SignonTest.java:73)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithWrong -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@4602c2a9]" started-at="2020-02-03T00:24:31Z" name="close" finished-at="2020-02-03T00:24:31Z" duration-ms="145" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.apex.ecommerce.signon.test.SignonTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
